### DISCUSSION TITLE
Folders for models

### DISCUSSION THREAD

**Discussion by Taikakim** [Comfy-Org/ComfyUI#1226] [Ideas]
I have a ton of LoRAs, it would be nice if the loader separated them according to directories. Now I spend a lot of time slowly scrolling up and down the list. Or even better, recognise names with numbers after them, and add a second drop down for model epoch variant selection.

**Comment by simonlui** [comment]
Have you taken a look at https://github.com/comfyanonymous/ComfyUI/blob/2c97c30256a00e71e93f11130012a5da0b726c17/extra_model_paths.yaml.example#L11-L13
You can see that there are two different folders for LoRA and LyCORIS and piggybacking off that file, you can list in YAML all the folders you want if you decide to seperate your LoRAs.

**Reply by Taikakim** [reply]
Hmm interesting! Thanks. I'll try to set them manually there. I wonder if every update on Colab will overwrite that?

**Reply by simonlui** [reply]
If it isn't overwriting your LoRAs, it shouldn't overwrite that file but I run locally so I have no clue.
